src/runtime/propensity.js

```diff
@@ -69,18 +89,19 @@ export class Propensity {
 
   /** @override */
   sendEvent(userEvent) {
-    if (!Object.values(PropensityApi.Event).includes(userEvent.name)) {
+    if (!isEnumValue(PropensityApi.Event, userEvent.name)
+        || !PropensityEventToAnalyticsEvent[userEvent.name]) {
       throw new Error('Invalid user event provided');
```

Comment:
Do you see any issues in augmenting the error message on line 94 to include the userEvent.name?

Reply:
I think its a good idea